### DISCUSSION TITLE
Toggle menu dropdown text align on the right

### DISCUSSION THREAD

**Discussion by kozzmen** [elementor/elementor#24634] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

As per Elementor Pro [documentation](https://elementor.com/help/nav-menu-widget-pro/) the mobile dropdown alignment has two options: Aside or Center. However there is no way to alight the menu items on the right hand site which is actually way more convenient to use, when you hold the phone with your right hand.  
![image](https://github.com/elementor/elementor/assets/9672040/957639dc-971b-4804-b256-ff08b97fd859)

### Describe the solution you'd like.

It would be great if the Aside option has the ability to choose between left and right alignment, as right now only left is available. 

### Describe alternatives you've considered.

Can anyone provide a quick fix with the help of CSS to achieve right alignment of the menu items?

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by lavendaniel** [comment]
This is really annoying me too, it looks hideous aligning text left when you've got the toggle on the right side of the screen. The mobile dropdown version of the menu is a total mess and doesn't have anywhere near enough styling control. If you don't have full width enabled, you can't control the width either, so have no choice but to have it full width.

**Reply by kozzmen** [reply]
I patched it with a few lines of CSS. 
If anyone needs it, here it is:

`/* Menu items - align right */
ul#menu-ID {
    display: flex;
    align-items: end;
    flex-direction: column;
}`

**Reply by lavendaniel** [reply]
Doesn't seem to work for me :(

**Reply by kozzmen** [reply]
Use the Inspect tool and find the proper ID, then substitute it in the code above. In my case it was something like: #menu-2-7e2d02.

**Reply by lavendaniel** [reply]
It turned my horizontal (desktop) menu into a vertical menu but didn't affect the dropdowns/mobile menu which is what I need right-aligned.

**Reply by lavendaniel** [reply]
Ooh actually that kinda works! Now I just need to get the width consistent
